influxdb3_write/src/write_buffer/loader.rs

```diff
@@ -37,6 +40,31 @@ where
     write_buffer::Error: From<<P as Persister>::Error>,
 {
     let PersistedCatalog { catalog, .. } = persister.load_catalog().await?.unwrap_or_default();
+
+    let last_cache = LastCacheProvider::new();
```

Comment:
I'm going to be removing the loader in my PR. Can you move the load/initialization code into the LastCacheProvider or some other thing that takes a Catalog and builds it from there?

Reply:
Sure, that shouldn't be an issue, will do!

Reply:
Moved in https://github.com/influxdata/influxdb/pull/25170/commits/77ecac346af3575104d999f0b0c07770fb7d9831

Also, I think there should be a test(s) for loading last cache state from the catalog. I filed https://github.com/influxdata/influxdb/issues/25183 to sort that out because I think this PR is big enough already.